docs/contributor/build-concierge.md

```diff
@@ -28,9 +28,9 @@ This will clean workspace, build and run tests, and will create a distribution f
 
 ## Versioning
 
-The framework and the bundles jars will be named to Bundle-SymbolicName and Bundle-Version, e.g. `org.eclipse.concierge-5.1.0.SNAPSHOT-20151114134106.jar`. The Manifest.MF will specify the `Bundle-Version: 5.1.0.qualifier`, qualifier will be replaced by the current build date/time (MANIFEST driven approach). The version maintenance is up to the individual bundles/framework dependent on compatibility of changes.
+The framework and the bundles jars will be named to Bundle-SymbolicName and Bundle-Version, e.g. `org.eclipse.concierge-0.9.0.20141230122000.jar`. The Manifest.MF will specify the `Bundle-Version: 0.9.0.qualifier`, qualifier will be replaced by the current build date/time (MANIFEST driven approach). The version maintenance is up to the individual bundles/framework dependent on compatibility of changes.
 
```

Comment:
the sample version should be better `5.1.0.SNAPSHOT-<current-timestamp>.jar`
Also the Bundle-Version should be `5.1.0.qualifier`